dynamic-demo-plugin/src/components/ClusterOverview/Inventory.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { MonitoringIcon } from '@patternfly/react-icons';
+import {
+  K8sResourceCommon,
+  useK8sWatchResource,
+  InventoryItem,
+  InventoryTitle,
+  InventoryBody,
+  InventoryStatus,
+  InventoryLoading,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const WorkerNodeInventory: React.FC<{}> = () => {
+  const { t } = useTranslation("plugin__console-demo-plugin");
+  const [nodes, loaded, loadError] = useK8sWatchResource<K8sResourceCommon[]>({
+    groupVersionKind: {
+      kind: 'Node',
+      version: 'v1',
+      group: 'core',
+    },
+    isList: true,
+  });
+
+  const workerNodes = nodes.filter((node) => !!node.metadata?.labels?.['node-role.kubernetes.io/worker']);
+
+  let title = <Link  to="/k8s/cluster/nodes">{t('{{count}} Worker Node', { count: nodes.length })}</Link>;
+  if (loadError) {
+    title = <Link to="/k8s/cluster/nodes">{t('Worker Nodes')}</Link>;
+  } else if (!loaded) {
+    title = <><InventoryLoading /><Link to="/k8s/cluster/nodes">{t('Worker Nodes')}</Link></>;
+  }
+
+  return (
+    <InventoryItem>
+      <InventoryTitle>{title}</InventoryTitle>
+      <InventoryBody error={loadError}>
+        {loaded && <InventoryStatus count={(workerNodes as any).length1.foo} icon={<MonitoringIcon />} />}
```

Comment:
```suggestion
        {loaded && <InventoryStatus count={(workerNodes as any[]).length.foo} icon={<MonitoringIcon />} />}
```

Reply:
good catch :D That should actually be just simple `count={workerNodes.length}`. It was a leftover when I was testing the error handling